build.gradle

```diff
@@ -116,6 +116,38 @@ subprojects {
             force "jdepend:jdepend:${project.jdepend.toolVersion}"
         }
     }
+
+    apply plugin: 'checkstyle'
+    configurations {
+        checkstyleConfig
+    }
+    def versions = [
+            checkstyle: '6.17',
+    ]
+    dependencies {
+        checkstyleConfig ("com.puppycrawl.tools:checkstyle:${versions.checkstyle}") {
+            transitive = false
+        }
+        checkstyle "com.puppycrawl.tools:checkstyle:${versions.checkstyle}"
+    }
+    checkstyle {
+        config = resources.text.fromArchiveEntry(configurations.checkstyleConfig, 'google_checks.xml')
+        ignoreFailures false
```

Comment:
I wonder if this can be the other way around: use "ignoreFailures false" and remove throwing an exception explicitly. This "ignoreFailures" worked nicely with PMD and FIndBugs.